### PR TITLE
Use local web service for test

### PR DESCRIPTION
**Pull request by jdietrich-tc**
Instead of relying on connectivity of `https://golang.org/` for testing http streaming, use a local web server, and call it. This makes the tests quicker and more reliable.

### CODE REVIEW

File: pkg/vars/stream_test.go [hunk 1]
```diff
@@ -43,17 +45,21 @@ func TestReadFile(t *testing.T) {
 
 func TestHTTPContent(t *testing.T) {
 	st := Stream{}
+	wg := &sync.WaitGroup{}
```
Comment (jmartin82) [inline pkg/vars/stream_test.go:48]: Hi, I'm struggeling to see why the WaitGroup is used. Could you elaborate a bit for me?
Thanks!

Comment (jdietrich-tc) [reply pkg/vars/stream_test.go:48]: It pauses the test until the http server is up and running. It communicates such when it calls the wait groups done function. 

Without it, there is a race condition between starting the server and attempting to use it.

Comment (jmartin82) [reply pkg/vars/stream_test.go:48]: But I don’t see any wg.Wait() that makes the test wait as you are describing. That's why a bit lost.

Comment (jdietrich-tc) [reply pkg/vars/stream_test.go:48]: To be honest, I'm not sure exactly how it works. I'm new to go (mmock is the first go code I've ever worked with) I found that I couldn't shut down the server once started, this hanging the tests. I found this, and it works. 

https://stackoverflow.com/questions/39320025/how-to-stop-http-listenandserve

**Comment by jdietrich-tc** [comment]
Closed in preference of https://github.com/jmartin82/mmock/pull/185

**Pull request closed without merging**